rbac/management/role/relation_api_dual_write_handler.py

```diff
@@ -57,44 +59,79 @@ def replication_enabled(self):
         """Check whether replication enabled."""
         return ENVIRONMENT.get_value("REPLICATION_TO_RELATION_ENABLED", default=False, cast=bool)
 
-    def generate_relations_from_current_state_of_role(self):
+    def get_current_role_relations(self):
```

Comment:
I reordered / renamed some methods in this class just for clarity while trying to understand how it was meant to be used.